SpatialGDK/Source/SpatialGDK/Private/Interop/ActorSystem.cpp

```diff
@@ -449,29 +449,10 @@ void ActorSystem::HandleActorAuthority(const Worker_EntityId EntityId, const Wor
 						Actor->SetReplicates(true);
 					}
 
-					if (Actor->IsA<APlayerController>())
+					if (Channel->IsAutonomousProxyOnAuthority())
```

Comment:
Oh - channel could be invalid here for dormant actors :(

Reply:
This function already accesses `Channel` above:
```
	if (Channel != nullptr)
	{
		if (ComponentSetId == SpatialConstants::SERVER_AUTH_COMPONENT_SET_ID)
		{
			Channel->SetServerAuthority(Authority == WORKER_AUTHORITY_AUTHORITATIVE);
		}
		else if (ComponentSetId == SpatialConstants::CLIENT_AUTH_COMPONENT_SET_ID)
		{
			Channel->SetClientAuthority(Authority == WORKER_AUTHORITY_AUTHORITATIVE);
		}
	}
```
If it could be invalid, wouldn't that also break?